wordpress-mobile/WordPress-Android
# Issue/8326 page upload notification

Fixes #8326.

To test:
1. Open a site that has many pages (so you have to scroll the list)
2. Open a page and edit it
3. Quickly exit the Site pages before the upload is finished (otherwise the notification isn't shown)
4. When the upload is done, a notification is shown
5. Tap the notification
6. Notice the Site pages screen is opened, scrolled so that the changed page is visible on the screen
